code/game/machinery/gashapon.dm

```diff
@@ -97,7 +97,7 @@
 		/obj/item/toy/gasha/cattoy,
 		/obj/item/toy/gasha/parrottoy,
 		/obj/item/toy/gasha/beartoy,
-		/obj/item/toy/gasha/carptoy,
+		/obj/item/toy/carpplushie,
```

Comment:
The space carp plushie toy comes with a sound, attack_verb and better sprite, I removed the gasha carptoy as it had the same purpose and was inferior